File: example/src/main/java/com/amqtech/opensource/appintroexample/ui/permsTabs/PermissionsLayout1Intro.java
package com.amqtech.opensource.appintroexample.ui.permsTabs;

import android.content.Intent;
import android.os.Bundle;
import androidx.annotation.Nullable;
import androidx.fragment.app.Fragment;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import android.widget.Button;

import com.amqtech.opensource.appintroexample.ui.permsTabs.intro.PermissionsIntro1;
import com.github.paolorotolo.appintroexample.R;

/**
 * Created by andrew on 11/17/16.
 */

public class PermissionsLayout1Intro extends Fragment {

    @SuppressWarnings("ConstantConditions")
    @Override
    public void onActivityCreated(@Nullable Bundle savedInstanceState) {
        super.onActivityCreated(savedInstanceState);

        Button demo = getView().findViewById(R.id.intro1);
        demo.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                startActivity(new Intent(getActivity().getBaseContext(), PermissionsIntro1.class));
            }
        });
    }

    @Nullable
    @Override
    public View onCreateView(LayoutInflater inflater, @Nullable ViewGroup container, @Nullable Bundle savedInstanceState) {
        return inflater.inflate(R.layout.tab_permissions_1, container, false);
    }
}

File: example/src/main/java/com/github/appintro/example/ui/permsTabs/PermissionsLayout1Intro.kt
package com.github.appintro.example.ui.permsTabs

import android.content.Intent
import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import androidx.fragment.app.Fragment
import com.github.appintro.example.ui.permsTabs.intro.PermissionsIntro1
import com.github.paolorotolo.appintroexample.R
import kotlinx.android.synthetic.main.tab_permissions_1.*

class PermissionsLayout1Intro : Fragment() {
    override fun onActivityCreated(savedInstanceState: Bundle?) {
        super.onActivityCreated(savedInstanceState)

        intro1.setOnClickListener {
            startActivity(Intent(context, PermissionsIntro1::class.java))
        }
    }

    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?,
                              savedInstanceState: Bundle?): View? =
            inflater.inflate(R.layout.tab_permissions_1, container, false)
}